report-pull-request-results.py

```diff
@@ -61,6 +61,7 @@ def parse_workflow_info( parts, relval_dir ):
   # it starts asumed as not found
   out_file = MATRIX_WORKFLOW_STEP_LOG_FILE_NOT_FOUND
   workflow_info[ 'step' ] = MATRIX_WORKFLOW_STEP_NA
+  out_directory = None
```

Comment:
this alone is not going to fix the issue. If `out_directory` is None then later `join` statement will fail.

Reply:
i'll try to get the conditions (arguments) of that function from the jenkins job then.

Reply:
no idea what you mean by that. Thsi function is reading the output of runTheMatrix and trying to find the failed workflow,  its directory path and step number. No idea how jenkins jobs is going to help you here

Reply:
looks like this job didn't fail because of the function but because the relvals were failing (I'll guess das queries)

Reply:
job failed as it was expecting one-line message like `ERROR executing  cd 4.4_RunMinBias2012A+RunMinBias2012A+HLTD+RECODR1reHLT+HARVESTDR1reHLT; ...`  but it found message on multiple line
```
ERROR executing  
cd 4.34_ValSkim2011B+ValSkim2011B+HLTDSKIM+RECODR1reHLT+HARVESTDR1reHLT; 
```
this causes to not find `cd` in the first line